packages/create-miniapp-cli/template/config/next-config-appdir.js

```
@@ -0,0 +1,10 @@
+/**
+ * Run `build` or `dev` with `SKIP_ENV_VALIDATION` to skip env validation. This is especially useful
+ * for Docker builds.
+ */
+import "./src/env.js";
+
+/** @type {import("next").NextConfig} */
+const config = {};
+
+export default config;
```

ESLint

[error] Extra semicolon.

(@typescript-eslint/semi)